src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/DocumentHelperTest.java

```diff
@@ -164,7 +162,84 @@ public void buildDocumentContent1() {
+                + "\"respondent_full_name\":\"Antonio Vazquez\",\n"
+                + "\"respondent_addressLine1\":\"11 Small Street\",\n"
+                + "\"respondent_addressLine2\":\"22 House\",\n"
+                + "\"respondent_addressLine3\":\"\",\n"
+                + "\"respondent_town\":\"Manchester\",\n"
+                + "\"respondent_county\":\"North West\",\n"
+                + "\"respondent_postCode\":\"M12 42R\",\n"
+                + "\"Respondent\":\"Antonio Vazquez\",\n"
+                + "\"resp_others\":\"\",\n"
+                + "\"resp_address\":\"\",\n"
+                + "\"Hearing_date\":\"\",\n"
+                + "\"Hearing_date_time\":\"\",\n"
+                + "\"Hearing_time\":\"\",\n"
+                + "\"Hearing_venue\":\"Manchester Employment Tribunals, Alexandra House, 14-22 The Parsonage, "
+                + "Manchester, M3 2JA\",\n"
+                + "\"Hearing_duration\":\"2 days\",\n"
+                + "\"t2_2A\":\"true\",\n"
+                + "\"Court_addressLine1\":\"Manchester Employment Tribunal,\",\n"
+                + "\"Court_addressLine2\":\"Alexandra House,\",\n"
+                + "\"Court_addressLine3\":\"14-22 The Parsonage,\",\n"
+                + "\"Court_town\":\"Manchester,\",\n"
+                + "\"Court_county\":\"\",\n"
+                + "\"Court_postCode\":\"M3 2JA\",\n"
+                + "\"Court_telephone\":\"03577131270\",\n"
+                + "\"Court_fax\":\"07577126570\",\n"
+                + "\"Court_DX\":\"123456\",\n"
+                + "\"Court_Email\":\"ManchesterOfficeET@hmcts.gov.uk\",\n"
+                + "\"i2_2A_enhmcts\":\"[userImage:enhmcts.png]\",\n"
+                + "\"i2_2A_enhmcts1\":\"[userImage:enhmcts.png]\",\n"
+                + "\"i2_2A_enhmcts2\":\"[userImage:enhmcts.png]\",\n"
+                + "\"iScot_schmcts\":\"[userImage:schmcts.png]\",\n"
+                + "\"iScot_schmcts1\":\"[userImage:schmcts.png]\",\n"
+                + "\"iScot_schmcts2\":\"[userImage:schmcts.png]\",\n"
+                + "\"Clerk\":\"Mike Jordan\",\n"
+                + "\"Today_date\":\"" + UtilHelper.formatCurrentDate(LocalDate.now()) + "\",\n"
+                + "\"TodayPlus28Days\":\"" + UtilHelper.formatCurrentDatePlusDays(LocalDate.now(), 28) + "\",\n"
+                + "\"Case_No\":\"123456\",\n"
+                + "}\n"
+                + "}\n";
+        caseDetails2.getCaseData().getRepCollection().get(0).getValue().setRespRepName("Antonio Vazquez");
+        assertEquals(expected, DocumentHelper.buildDocumentContent(caseDetails2.getCaseData(), "",
+                userDetails, MANCHESTER_CASE_TYPE_ID, venueAddressInputStream,
+                caseDetails2.getCaseData().getCorrespondenceType(),
+                caseDetails2.getCaseData().getCorrespondenceScotType(), null,
+                null).toString());
+        caseDetails2.getCaseData().getRepCollection().get(0).getValue().setRespRepName("RepresentativeNameRespondent");
```

Comment:
Does this line serve any purpose?

Reply:
Unfortunately other tests fail, if this line is not there. We have to reset the RespRepName to the initial value as it was in CaseDetailsTest2.json.